web_monitoring/db.py

```diff
@@ -725,7 +734,7 @@ def get_version_content(self, version_id):
         """
         db_result = self.get_version(version_id)
         content_uri = db_result['data']['uri']
-        res = requests.get(content_uri)
+        res = self.get(content_uri)
         _process_errors(res)
         if res.headers.get('Content-Type', '').startswith('text/'):
```

Comment:
Now that we're setting the accept header to json on all requests, will this branch ever be hit?

Reply:
Ohhhhh, thanks for catching this! We actually shouldn’t be setting the `accepts` header for this (though I’m not sure it will make a difference in most cases) and yes, this branch should still be here.

This method is sort of an odd one out — it’s getting the raw response body from an archived request rather than data (i.e. it returns the response body associated with a particular Version), so we could get back all kinds of stuff. The branch here is meant to be a useful shortcut (since `.content` is bytes and `.text` is a decoded string).

Reply:
Posted a change that restores this special case and blanks out the accept header (as per the "Remove a value from a from a dict parameter" note [here](https://requests.kennethreitz.org/en/master/user/advanced/#session-objects). How's that look to you?

Reply:
Makes a lot of sense! The one thing that feels a bit iffy about it to me is the `raw_response` parameter, which changes the return type. Somebody once suggested I should avoid writing functions where an argument switches the return types or shape, which I’ve generally found to be smart advice. (So I might just use the session directly in the one exceptional situation, or have a `request` and a `request_json` method that adds the parsing to `request` or something.)

That said, I don’t think this is an especially big deal in this one case. Happy to merge as is.

Reply:
No, you're right - I (and Python) let myself do the easy thing. I split the core logic into a request method and added a request_json wrapper that all but this instance use.